core/controllers/library.py

```diff
@@ -149,7 +149,8 @@ def get(self):
 
 
 class LibraryGroupPage(base.BaseHandler):
-    """The page for displaying top rated and recently published explorations.
+    """The page for displaying top rated and recently published
+    explorations.
```

Comment:
Again, the word `explorations` has been moved to the next line. The original sentence length here is 73.